core/domain/draft_upgrade_services_test.py

```diff
@@ -119,6 +119,19 @@ def test_convert_to_latest_schema_version_implemented(self):
             msg='Current schema version is %d but DraftUpgradeUtil.%s is '
             'unimplemented.' % (state_schema_version, conversion_fn_name))
 
+    def test_convert_states_v30_dict_to_v31_dict(self):
+        draft_change_list = [
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': 'Intro',
+                'property_name': 'content',
+                'new_value': 'new value'
+            })]
+        self.assertEqual(
+            draft_upgrade_services.DraftUpgradeUtil._convert_states_v30_dict_to_v31_dict(  # pylint: disable=protected-access,line-too-long
```

Comment:
Break it into 2 lines instead of using pragma. 

```
draft_upgrade_services.DraftUpgradeUtil
._convert_states_v30_dict_to_v31_dict(...
```

Reply:
Ok, should I do it for all of the other methods in file like _convert_states_v29_dict_to_v30_dict ?

Reply:
done.